shell/config/router.js

```diff
@@ -1,506 +0,0 @@
-        }, {
-          path:      '/c/:cluster/monitoring/monitor/:namespace/:id?',
-          component: () => interopDefault(import('../pages/c/_cluster/monitoring/monitor/_namespace/_id.vue')),
-          name:      'c-cluster-monitoring-monitor-namespace-id'
-        }, {
-          path:      '/c/:cluster/navlinks/:group?',
-          component: () => interopDefault(import('../pages/c/_cluster/navlinks/_group.vue')),
-          name:      'c-cluster-navlinks-group'
-        }, {
-          path:      '/c/:cluster/:product',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/index.vue')),
-          name:      'c-cluster-product'
-        }, {
-          path:      '/c/:cluster/:product/members',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/members/index.vue')),
-          name:      'c-cluster-product-members'
-        }, {
-          path:      '/c/:cluster/:product/namespaces',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/namespaces.vue')),
-          name:      'c-cluster-product-namespaces'
-        }, {
-          path:      '/c/:cluster/:product/projectsnamespaces',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/projectsnamespaces.vue')),
-          name:      'c-cluster-product-projectsnamespaces'
-        }, {
-          path:      '/c/:cluster/:product/:resource',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/_resource/index.vue')),
-          name:      'c-cluster-product-resource'
-        }, {
-          path:      '/c/:cluster/:product/:resource/create',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/_resource/create.vue')),
-          name:      'c-cluster-product-resource-create'
-        }, {
-          path:      '/c/:cluster/:product/:resource/:id',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/_resource/_id.vue')),
-          name:      'c-cluster-product-resource-id'
-        }, {
-          path:      '/c/:cluster/:product/:resource/:namespace/:id?',
-          component: () => interopDefault(import('../pages/c/_cluster/_product/_resource/_namespace/_id.vue')),
-          name:      'c-cluster-product-resource-namespace-id'
-        }]
-    }],
-
-  fallback: false
-};
-
-export function createRouter(config) {
-  const base = (config._app && config._app.basePath) || routerOptions.base;
-  const router = new Router({ ...routerOptions, base });
-
-  // TODO: remove in Nuxt 3
-  const originalPush = router.push;
-
-  router.push = function push(location, onComplete = emptyFn, onAbort) {
-    return originalPush.call(this, location, onComplete, onAbort);
-  };
-
-  const resolve = router.resolve.bind(router);
-
-  router.resolve = (to, current, append) => {
-    if (typeof to === 'string') {
-      to = normalizeURL(to);
-    }
-
-    return resolve(to, current, append);
-  };
```

Comment:
I removed this code. It's possible this may cause a regression but I think it's unlikely. I didn't find any obvious places where normalizeURL would make a difference.